ocppj/ocppj.go

```diff
@@ -387,7 +398,7 @@ func (endpoint *Endpoint) ParseMessage(arr []interface{}, pendingRequestState Cl
 	typeId := MessageType(rawTypeId)
 	uniqueId, ok := arr[1].(string)
 	if !ok {
-		return nil, ocpp.NewError(FormationViolation, fmt.Sprintf("Invalid element %v at 1, expected unique ID (string)", arr[1]), uniqueId)
```

Comment:
Should be `""` for better readability, since `uniqueId` is `""` in this case

Reply:
Yes, good point